tests/test_tasks.py

```diff
@@ -2952,6 +2952,32 @@ def my_flow():
         task_states = my_flow()
         assert [state.result() for state in task_states] == [6, 7, 8]
 
+    def test_with_variadic_keywords_and_iterable(self):
+        @task
+        def add_some(x, **kwargs):
+            return x + kwargs["y"]
+
+        @flow
+        def my_flow():
+            numbers = [1, 2, 3]
+            return add_some.map(numbers, y=[4, 5, 6])
+
+        task_states = my_flow()
+        assert [state.result() for state in task_states] == [5, 7, 9]
+
+    def test_with_variadic_keywords_and_noniterable(self):
+        @task
+        def add_some(x, **kwargs):
+            return x + kwargs["y"]
+
+        @flow
+        def my_flow():
+            numbers = [1, 2, 3]
+            return add_some.map(numbers, y=1)
```

Comment:
I think this may need to be:
```python
            return add_some.map(numbers, y=unmapped(1))
```


...where `unmapped` can be imported directly from the `prefect` module.
I'm not familiar enough with the codebase to know how to test that quickly, though.

Reply:
Nope! We detect `unmapped` automatically now. It can be used for iterable types that you don't want to map over, but isn't needed otherwise.

Reply:
Oh nice!